docs/en/OLED_Display.md

```diff
@@ -0,0 +1,199 @@
+# Oled Display
```

Comment:
```suggestion
# OLED Display
```
Use consistend spelling, *OLED* instead of *Oled*.
I noticed that the extension also calls it's main class *Oled*, and we should fix that before we publish the documentation.